onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.cc

```diff
@@ -736,7 +736,15 @@ bool ConvOpSupportChecker::IsOpSupportedImpl(const InitializedTensorSet& initial
   NodeAttrHelper helper(node);
   size_t w_idx = is_qlinear_conv ? 3 : 1;
   const auto group = helper.Get("group", 1);
-  const auto weight_name = input_defs[w_idx]->Name();
+
+  std::string weight_name;
+  if (params.qdq_support_helper->IsNodeInQDQGroup(node)) {
+    const auto* dq_w = params.qdq_support_helper->dq_nodes_in_qdq_selection.at(1);
+    weight_name = dq_w->InputDefs()[0]->Name();
```

Comment:
Should this attempt to retrieve the QDQ NodeGroup for the current node, and if found, use DQ/Q info from that?

Reply:
Will change to that.